### PR TITLE
Sunset Prometheus tutorial

### PR DESCRIPTION
**Pull request by cevian**
This tutorial used the old prometheus adapter, which
is getting sunsetted.

### CODE REVIEW

**Comment by mfreed** [comment]
I somewhat agree with @solugebefola here.  Much like we aren't *taking down* the github repos of the old adaptor, perhaps this content should still exist somewhere, but just got linked and with a warning?

**Comment by mfreed** [comment]
@cevian Thoughts?

**Comment by mfreed** [comment]
Anything keeping this from merge @cevian ?

File: tutorials/tutorial-setup-timescale-prometheus.md [hunk 1]
```diff
@@ -1,3 +1,13 @@
+# Sunset Notice
+
+This page contain a tutorial for an deprecated method of storing Prometheus data in
```
Comment (svenklemm) [inline tutorials/tutorial-setup-timescale-prometheus.md:3]:
```suggestion
This page contains a tutorial for a deprecated method of storing Prometheus data in
```

**Pull request merged**